packages/devextreme/js/ui/chat.d.ts

```diff
@@ -168,6 +168,30 @@ export interface dxChatOptions extends WidgetOptions<dxChat> {
      * @public
      */
     errors?: Array<ChatError>;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    showDayHeaders?: boolean;
```

Comment:
We need to remove 'showDayHeaders' member from our internal types in Chat.ts file.

Reply:
[2abb661](https://github.com/DevExpress/DevExtreme/pull/28219/commits/2abb6612a7314f858e6be00e9e1ecab2b0f40194)